README.md

```diff
@@ -58,3 +59,59 @@ ninja -C out/Ozone mus_ws_unittests mus_demo_unittests
 
 Note that `--ozone-platform` can be passed to all the programs above to select
 a specific Ozone backend.
+
+# Rebase Strategy
+
+As mentioned above, the fork is rebased every week against Chromium ToT.
+The goal is to be as close as possible to the lastest Mus code, which is
+constantly receiving perform and stability fixes.
+
+Here is the currect process:
+
+* Every week, a member of the Igalia Chromium team takes the rebase shift.
+
+* Commits that are complementary of each other, receive a "fixup!" prefix on
+the commit title, and keep the rest of original commit title unchanged.
+
+For example:
+
+```
+$ git log --oneline
+commit 1
+commit 2
+commit 3
+fixup! commit 1
+fixup! commit 2
+commit 4
+fixup! commit 2
+(..)
+```
+
+This allows an easy identification of "fixup" commits, which should be squashed into
+their original counterpart commit as part of the next rebase cycle. That way we keep
+our Git history clean, and commits as atomic as possible, for when upstreaming.
+
+Git has an optimized flow for this as well: http://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html
+
+* We always keep the 'ozone-wayland-dev' branch as our primarily development branch.
+
+This means that force pushes will happen. So every time one of the team members
+rebases our branch, the developer should first back up the existing ozone-wayland-dev
+browser, with the following naming: ozone-wayland-dev-rXXXX, where XXXX is the respective
+Chromium baseline of the branch.
+
+* Branch acceptance criteria
+
+compilation targets:
+
+```
+chrome mash:all  mus_ws_unittests  base_unittests services/ui/demo mus_*_unittests ozone_unittests mus_demo_unittests
+```
+... on LinuxOS and ChromeOS builds.
+
+Pass `mus_demo` and `mus_demo_unittests` tests.
+
+Run chrome --mash / --mus on the configurations above, and performance some sanity tests:
+basic mouse clicking / keyboard; basic navigation; multi window creation; etc.
+
+* Keep our internal buildbot green.
```

Comment:
Yes, I agree with Maksim, the buildbot link should be provided. Actually, maybe you can provide a reference to the targets / tests executed on the bot so that you do not need to repeat them here.